.github/workflows/links.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   check-links:
     runs-on: ubuntu-latest
     permissions:
-      contents: read
+      issues: write # required for creating issues from link checker reports
```

Comment:
is this issue creation working currently?

I'm wondering because this file already had root-level permissions block:

```
permissions: read-all
```

and so I think this job is only inheriting `read-all` permissions currently

Reply:
Would `permissions: read-all` be good enough? Why do we need to set additional permissions?